public/app/features/serviceaccounts/CreateTokenModal.tsx

```diff
@@ -0,0 +1,161 @@
+import React, { useState } from 'react';
+import { css } from '@emotion/css';
+import { GrafanaTheme2 } from '@grafana/data';
+import {
+  Button,
+  ClipboardButton,
+  DatePickerWithInput,
+  Field,
+  FieldSet,
+  HorizontalGroup,
+  Icon,
+  Input,
+  Label,
+  Modal,
+  RadioButtonGroup,
+  useStyles2,
+} from '@grafana/ui';
+
+const expirationOptions = [
```

Comment:
```suggestion
const EXPIRATION_OPTIONS = [
```

ref https://github.com/grafana/grafana/blob/main/contribute/style-guides/frontend.md#use-all_caps-for-constants